package.json

```diff
@@ -36,7 +36,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "prettier": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'",
+    "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md,scss,yaml,yml}\"",
```

Comment:
By keeping the single quotes you could avoid having to escape them.

Reply:
Done